From e7cecb382163b727ae0e20ad0a3e6237f7fa7d83 Mon Sep 17 00:00:00 2001
From: Gayathri <gathreya@email.arizona.edu>
Date: Fri, 29 May 2015 18:49:07 -0700
Subject: [PATCH] RESKC-491: Modified parent annotation for action request to
 display people flow name. For the KRMS Agenda based routing approval stops,
 PPL Flow Member "Annotation" is still blank and does not display any PPL Flow
 info to user when they open the Future routing.

Only the delegates, visible only once the show is expanded, have the PPL flow name included.

PPL Flow Member Annotation field in Routing Log needs to show PPL Flow Name. User needs to be able to see PPL Flow names for each Pending Approval stop without expanding to display delegates.
---
 .../kuali/rice/kew/impl/peopleflow/PeopleFlowRequestGeneratorImpl.java  | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rice-middleware/impl/src/main/java/org/kuali/rice/kew/impl/peopleflow/PeopleFlowRequestGeneratorImpl.java b/rice-middleware/impl/src/main/java/org/kuali/rice/kew/impl/peopleflow/PeopleFlowRequestGeneratorImpl.java
index a9829f8..158d5093 100644
--- a/rice-middleware/impl/src/main/java/org/kuali/rice/kew/impl/peopleflow/PeopleFlowRequestGeneratorImpl.java
+++ b/rice-middleware/impl/src/main/java/org/kuali/rice/kew/impl/peopleflow/PeopleFlowRequestGeneratorImpl.java
@@ -92,6 +92,8 @@ protected void generateRequestForMember(Context context, PeopleFlowMember member
             if (actionRequest != null) {
                 generateDelegationRequests(context, Collections.singletonList(actionRequest), member);
             }
+
+            actionRequest.setAnnotation(PEOPLE_FLOW_NAME + context.getPeopleFlow().getName() + System.getProperty("line.separator"));
         }
     }
 
